Doc/whatsnew/3.11.rst

```diff
@@ -131,41 +131,36 @@ as well as complex arithmetic expressions:
                   ~~~~~~^~~
     ZeroDivisionError: division by zero
 
-See :pep:`657` for more details. (Contributed by Pablo Galindo, Batuhan Taskaya
-and Ammar Askar in :issue:`43950`.)
-
-.. note::
-   This feature requires storing column positions in code objects which may
-   result in a small increase of disk usage of compiled Python files or
-   interpreter memory usage. To avoid storing the extra information and/or
-   deactivate printing the extra traceback information, the
-   :option:`-X` ``no_debug_ranges`` command line flag or the :envvar:`PYTHONNODEBUGRANGES`
-   environment variable can be used.
-
-Column information for code objects
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The information used by the enhanced traceback feature is made available as a
-general API that can be used to correlate bytecode instructions with source
-code. This information can be retrieved using:
+Additionally, the information used by the enhanced traceback feature
+is made available via a general API, that can be used to correlate
+:term:`bytecode` :ref:`instructions <bytecodes>` with source code location.
+This information can be retrieved using:
 
 - The :meth:`codeobject.co_positions` method in Python.
-- The :c:func:`PyCode_Addr2Location` function in the C-API.
-
-The :option:`-X` ``no_debug_ranges`` option and the environment variable
-:envvar:`PYTHONNODEBUGRANGES` can be used to disable this feature.
+- The :c:func:`PyCode_Addr2Location` function in the C API.
 
 See :pep:`657` for more details. (Contributed by Pablo Galindo, Batuhan Taskaya
 and Ammar Askar in :issue:`43950`.)
 
-Exceptions can be enriched with notes (PEP 678)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The :meth:`add_note` method was added to :exc:`BaseException`. It can be
-used to enrich exceptions with context information which is not available
-at the time when the exception is raised. The notes added appear in the
-default traceback. See :pep:`678` for more details. (Contributed by
-Irit Katriel in :issue:`45607`.)
+.. note::
+   This feature requires storing column positions in :ref:`codeobjects`,
+   which may result in a small increase in interpreter memory usage
+   and disk usage for compiled Python files.
+   To avoid storing the extra information
+   and deactivate printing the extra traceback information,
+   use the :option:`-X no_debug_ranges <-X>` command line option
+   or the :envvar:`PYTHONNODEBUGRANGES` environment variable.
+
+
+Exceptions can be enriched with notes
+-------------------------------------
+
+The :meth:`~BaseException.add_note` method was added to :exc:`BaseException`.
+It can be used to enrich exceptions with context information
+that is not available at the time when the exception is raised.
+The added notes appear in the default traceback.
+See :pep:`678` for more details.
+(Contributed by Irit Katriel in :issue:`45607`.)
```

Comment:
```suggestion
(Contributed by Irit Katriel in :gh:`89770`.)
```

Reply:
I thought about this, but I didn't end up updating these, since the BPO issue was canonical at the time the work was done, and the links redirect to the up to date GH issue. Is there any guidance on whether old `:issue:` refs should be manually converted, or left as-is? I couldn't find any in the devguide; if not, it might be better to make this a separate issue/discussion and be done consistently throughout.

Reply:
IMO it's best to avoid links to redirects, but it's not a big deal.

Reply:
Yeah, if people agree I'm happy to go through and convert them; I just think its best to do it in a separate PR after these more important ones are all merged, so it can be decided there, its consistent and we don't hold up the rest of the changes since we're on the clock here.